Composer/packages/lib/indexers/src/utils/dialogCheckUtil.ts

```diff
@@ -13,7 +13,7 @@ export function autofixReferInDialog(dialogId: string, content: string): string
 
     // fix lu referrence
     if (typeof dialogJson.recognizer === 'string') {
-      dialogJson.recognizer = `${dialogId}.lu`;
+      dialogJson.recognizer = `${dialogId.toLowerCase()}.lu.qna`;
```

Comment:
Latest policy is changed to allow user create uppercase Dialog, then I guess for `QnA` is same

Reply:
Another question here is why we name it `.lu.qna` instead of  `.qna` ?

Reply:
I'll delete the `toLowerCase` function, And the reason for using `.lu.qna` is that the recognizer is crosstrain recognizer involving qna & lu